src/components/menus/content-contextual-menu.tsx

```diff
@@ -247,7 +247,7 @@ const ContentContextualMenu = (props: ContentContextualMenuProps) => {
                         activeElement.elementUuid,
                         undefined,
                         activeElement.type,
-                        selectedElements[0].specificMetadata.type
+                        selectedElements[0].specificMetadata.type as unknown as string
```

Comment:
you cast a record to a string?!
```suggestion
                        // @ts-expect-error TODO: seems to be an object but we await a string???
                        selectedElements[0].specificMetadata.type
```

Reply:
In this case `specificMetadata` is defined as an object as a lack of a better definition since its content can be inconsistent, I was casting `type` to a string since I know it will always be a string

Reply:
Either the type is fixed, or the usage is noted "todo fix type" I would said